.github/workflows/pull-request.yml

```
@@ -78,10 +89,10 @@
           rm -rf private-repository
           git clone git@gitlab.com:MrSquaare/private-repository.git
 
-      - name: Run tests
+      - name: Run tests for GitLab
         env:
-          OUTPUT_SSH_PATH: ${{ steps.setup.outputs.ssh-path }}
-          OUTPUT_SSH_AGENT_PID: ${{ steps.setup.outputs.ssh-agent-pid }}
-          OUTPUT_SSH_AUTH_SOCK: ${{ steps.setup.outputs.ssh-auth-sock }}
+          OUTPUT_SSH_PATH: ${{ steps.gitlab.outputs.ssh-path }}
```

actionlint (1.7.7)

property "ssh-path" is not defined in object type {}

(expression)